docs/data/charts/pie-demo/PieChartWithCenterLabel.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { PieChart } from '@mui/x-charts/PieChart';
-import { useDrawingArea } from '@mui/x-charts/hooks/useDrawingArea';
```

Comment:
This code is in the `docs` folder. I don't think that `eslint` targets it. 🤔

Reply:
![image](https://github.com/mui/mui-x/assets/3309670/8e775c4f-7662-40b7-ab92-81e758ccad30)

TreeView imports are restricted in the doc.
I don't know which line of the lint prevents it though because I don't find any restricted import targeting the doc in our lint config.